### ISSUE TITLE
Metric for cluster_healthy (single cluster check)

### ISSUE THREAD

**Issue by mat1010** [Napsty/check_rancher2#12]
Hi,

https://github.com/Napsty/check_rancher2/blob/64d534956763a7933e48e4ac2c42df3316488047/check_rancher2.sh#L225

In the above mentioned line, the metric for `cluster_healhy`, in case a single cluster is checked, is always 0, even if the cluster is healthy. Wouldn't it be useful to hardcode it to 1 instead, in case there are no `componenterrors` ?

Best,
mat1010

**Comment by Napsty** [comment]
Good catch. Can confirm.
On multiple clusters the correct number is shown:

```
$ ./check_rancher2.sh -H rancher2api -U token-XXXXX -P "secret" -S -t cluster
CHECK_RANCHER2 OK - All clusters (2) are healthy|'clusters_total'=2;;;; 'clusters_errors'=0;;;;
```

But a single cluster check results in 0:

```
$ ./check_rancher2.sh -H rancher2api -U token-XXXXX -P "secret" -S -t cluster -c c-s2c8b
CHECK_RANCHER2 OK - Cluster c-s2c8b is healthy|'cluster_healthy'=0;;;; 'cluster_errors'=0;;;;
```

**Comment by Napsty** [comment]
Fixed in #13
